Robot_Adapter/Read/Elements/Bar.cs

```diff
@@ -59,8 +59,9 @@ private List<Bar> ReadBars(List<string> ids = null)
             HashSet<string> tags = new HashSet<string>();
 
             m_RobotApplication.Project.Structure.Bars.BeginMultiOperation();
-            if (ids == null)
+            if (barIds == null)
```

Comment:
Think this now needs to check if the list is empty as well:

```
if (barIds == null || barIds.Count < 0)
```

Reply:
Makes sense!